smoke-tests/src/test/java/io/opentelemetry/smoketest/windows/WindowsTestContainerManager.java

```diff
@@ -134,8 +132,13 @@ public int getTargetMappedPort(int originalPort) {
       String jvmArgsEnvVarName,
       Map<String, String> extraEnv,
       List<ResourceMapping> extraResources,
+      List<Integer> extraPorts,
       TargetWaitStrategy waitStrategy,
       String[] cmd) {
+    if (extraPorts != null && !extraPorts.isEmpty()) {
+      throw new UnsupportedOperationException("extra ports not supported");
```

Comment:
what was the issue with passing it to `withExposedPorts()` below?

Reply:
There is also the `withPortBindings` part. I don't have an easy way to test on windows so I chose to leave it unimplemented instead of adding something that might not work. This test doesn't run on windows anyway.